### DISCUSSION TITLE
🌎 Visitor Traffic Real Time Statistics --- Plugin Integration

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5675] [New Feature/Ideas Requests]
One very important feature of online e-commerce website is to know from where a customer is visiting. It is very important for an admin to know where a customer is visiting from and whether he is visiting from a browser or mobile. Or an admin needs to know how many visits a customer is getting from mobile apps. Then it will be convenient for an admin to do marketing later by getting more visitors from all his areas. Daily Weekly, real time how many people are visiting. WordPress has many plugins with real time traffic. Please use the plugin that is convenient for you to integrate. Free is much better if the plugin is free, Or maximum features can be used for free, please use this type of plugin. 

Please look into the matter seriously. 

WordPress real time traffic.... plugin list
https://wordpress.org/plugins/search/Visitor+Traffic+Real+Time+Statistics/

**Comment by osman-kocak** [comment]
Its very important
